sys/src/cmd/winwatch.c

```diff
@@ -277,8 +276,11 @@ main(int argc, char **argv)
 	lightblue = allocimagemix(display, DPalebluegreen, DWhite);
 	if(lightblue == nil)
 		sysfatal("allocimagemix: %r");
-	if((font = openfont(display, fontname)) == nil)
-		sysfatal("font '%s' not found", fontname);
+	if(fontname == nil)
+		font = opendefaultfont(display);
```

Comment:
Because that solves this particular case only. In programs in what there's no any -f font flag, we should include always a nil char *name just for that. Or a font flag. The main purpose of opendefaultfont func was avoiding the cases of not using the default, no any font specified or it's harcoded. A name second argument would be duplicating the openfont func and just for this specifical case in this program.
The simpler solution for winwatch shouldn't be one line for check the font, but removing font flag at all. Because seeing the purpose of the app, choosing font hasn't any sense here.
Agree?